docs/patterns/labels.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: default
+---
+
+## Labels
+
+<hr>
+
+Labels are static elements which you can apply to HTML elements to signify status, tags or any other information you find useful.
```

Comment:
I don't think you need this part of the sentence "to HTML elements"

Reply:
Fixed 👍